.github/workflows/sycl-linux-precommit.yml

```
@@ -236,6 +237,57 @@ jobs:
       skip_run: ${{matrix.use_igc_dev && contains(github.event.pull_request.labels.*.name, 'ci-no-devigc') || matrix.skip_run || 'false'}}
       env: ${{ matrix.env || (contains(needs.detect_changes.outputs.filters, 'esimd') && '{}' || '{"LIT_FILTER_OUT":"ESIMD/"}') }}
 
+  E2E-with-new-offload-model:
+    needs: [build, detect_changes, compat_read_exclude]
+    if: |
+      !cancelled() &&
+      needs.build.outputs.build_conclusion == 'success' &&
+      contains(github.event.pull_request.labels.*.name, 'new-offload-model')
+    permissions:
+      contents: write
+      packages: read
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Intel / GEN 12 Integrated
+            runner: '["Linux", "gen12"]'
+            target_devices: opencl:cpu;level_zero:gpu
+          - name: Intel / Arc A-Series Graphics
+            runner: '["Linux", "arc"]'
+            target_devices: opencl:cpu;level_zero:gpu
+          - name: Intel / Battlemage Graphics
+            runner: '["Linux", "bmg"]'
+            target_devices: level_zero_v2:gpu
+          - name: Intel / Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            target_devices: opencl:cpu;level_zero:gpu
```

Please, don't add cpu for each HW in precommit, it is enough to have it tested on one system, similar to general E2E.
Why not just use what I proposed above? Do you have any concerns or arguments?

```suggestion
          - name: Intel / GEN 12 Integrated
            runner: '["Linux", "gen12"]'
            target_devices: opencl:gpu;opencl:cpu
          - name: Intel / Arc A-Series Graphics
            runner: '["Linux", "arc"]'
            target_devices: level_zero:gpu
          - name: Intel / Ponte Vecchio GPU
            runner: '["Linux", "pvc"]'
            target_devices: level_zero:gpu
          - name: Intel / Battlemage Graphics
            runner: '["Linux", "bmg"]'
            target_devices: level_zero_v2:gpu
```